### DISCUSSION TITLE
How to adjust the font size and margin of content？

### DISCUSSION THREAD

**Discussion by CodeLife2012** [mkdocs/mkdocs#2837] [Q&A]
I use [mkdocs-pdf-export-plugin](https://github.com/zhaoterryy/mkdocs-pdf-export-plugin) to export some docs as pdf, and want to change the default font size and content margin.

Change content area margin seems not work.
https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#content-area-width

**Comment by pawamoy** [comment]
Have you tried to ask the same question on mkdocs-pdf-export-plugin's bugtracker?
